Add documentation for HomeWizard battery group power (#45588)
home-assistant/home-assistant.github.io: source/_integrations/homewizard.markdown

To learn how to update your device to the latest version, see [How do I check if I have the latest software on my HomeWizard product?](https://helpdesk.homewizard.com/en/articles/9167578-how-do-i-check-if-i-have-the-latest-software-on-my-homewizard-product)
{% endtip %}

> #### Battery group power and battery group target power

Reviewer: ```suggestion
#### Battery group power and battery group target power

```

> The **Battery group power** entity shows the total power consumed or produced by the group of connected Plug-In Batteries. The **Battery group target power** entity can be used to get the target power determined by the current battery group charging strategy. This is the power that the battery will try to achieve by charging or discharging. The value is negative when the battery is trying to produce power, and positive when it is trying to consume power.

Reviewer: ```suggestion
The **Battery group power** entity shows the total power consumed or produced by the group of connected Plug-In Batteries. The **Battery group target power** entity can be used to get the target power determined by the current battery group charging strategy. This is the power the battery group aims to deliver during charging or discharging. The value is negative when the battery group is trying to produce power, and positive when it is trying to consume power.
```
I added "group" there, but I'm not 100% sure this is correct. Could you take a look?

Author: Yes that is right, good catch!
